From 6b8650a2e10a00088b977467bf0c79dd7044fcd9 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Mon, 10 Nov 2014 17:00:20 +0100
Subject: [PATCH] cgroups-agent: really down-grade log level

(rhel-only, fix for broken backport b0010d76a8c56900d15fab8ac1ed94d364d1e566)

Related: #1044386
---
 src/cgroups-agent/cgroups-agent.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/cgroups-agent/cgroups-agent.c b/src/cgroups-agent/cgroups-agent.c
index 43b9b3a..57db147 100644
--- a/src/cgroups-agent/cgroups-agent.c
+++ b/src/cgroups-agent/cgroups-agent.c
@@ -50,11 +50,10 @@ int main(int argc, char *argv[]) {
 
         bus = dbus_connection_open_private("unix:path=/run/systemd/private", &error);
         if (!bus) {
-                log_warning("Failed to get D-Bus connection: %s", strerror(-r));
                 /* If we couldn't connect we assume this was triggered
                  * while systemd got restarted/transitioned from
                  * initrd to the system, so let's ignore this */
-                log_debug("Failed to get D-Bus connection: %s", strerror(-r));
+                log_debug("Failed to get D-Bus connection: %s", bus_error_message(&error));
                 goto finish;
         }
 
